openedx/features/course_experience/tests/views/test_course_home.py

```diff
@@ -92,6 +101,15 @@ def setUpTestData(cls):
         cls.user = UserFactory(password=TEST_PASSWORD)
         CourseEnrollment.enroll(cls.user, cls.course.id)
 
+    def create_future_course(self, specific_date=None):
```

Comment:
This is a really nice clean up of these tests. This method in particular makes the tests much easier to understand.